crates/ide-db/src/rename.rs

```diff
@@ -81,7 +81,7 @@ impl Definition {
             // Can we not rename non-local items?
             // Then bail if non-local
             if !rename_external && !krate.origin(sema.db).is_local() {
-                bail!("Cannot rename a non-local definition. Set `renameExternalItems` to `true` to allow renaming for this item.")
+                bail!("Cannot rename a non-local definition. Set `rename_allowExternalItems` to `true` to allow renaming for this item.")
```

Comment:
```suggestion
                bail!("Cannot rename a non-local definition as the config for it is disabled")
```
The config will be somewhat editor dependent so we should be more general